blocks/form/form.js

```diff
@@ -20,12 +21,16 @@ function constructPayload(form) {
   const payload = {};
```

Comment:
@Pareesh  payload should be initialized with unique id as in afb : https://github.com/adobe/afb/blob/main/blocks/form/form.js#L24